app/views/user/show.html.erb

```diff
@@ -1,15 +1,45 @@
-<h1> Users </h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blog App</title>
+    <link rel="stylesheet" href="application.css">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">   
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=5.7.0/css/font-awesome.min.css">     
+     
+    
+</head>
+<body>
+<div class='user'>
+<div class='user name'>
+<img class='user img' src='<%= @user.photo %>'>
+<h1><%= @user.name %></h1>
+</div>
+</div>
+<p class='add user'><%= link_to "Add new User", {:action => 'new'} %></p>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update this button styling as it marks this page as not looking as expected.